### PR TITLE
US137692 -- When the rubrics working copy flag is enabled, generate a working copy for rubrics

### PR DESCRIPTION
**Pull request by mpharoah-d2l**
https://rally1.rallydev.com/#/57732444928d/custom/373260458992?detail=%2Fuserstory%2F630919185551

This change will make rubrics use working copy when the flag is enabled. The Update/Publish button is not yet hooked up, so the rubric will not actually save yet, and only the inline view currently works with working copy (popout is still autosave). Both of these things will be addressed in a future pull request. The flag is disabled by default in dev, so this will not break rubrics for anyone. The feature flag is `rubrics-working-copy`.

### CODE REVIEW

File: components/consistent-evaluation.js [hunk 1]
```diff
@@ -365,6 +373,24 @@ export class ConsistentEvaluation extends LocalizeConsistentEvaluation(LitElemen
 		}
 	}
 
+	_makeUuid() {
```
Comment (JamieTran) [inline components/consistent-evaluation.js:376]: Correct me if I'm wrong but if this is just generating a uniqueID could we just can't we use something like https://github.com/BrightspaceUI/core/blob/main/helpers/uniqueId.js like we use in the [right-panel](https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/blob/master/components/right-panel/consistent-evaluation-right-panel.js#L277)

Comment (mpharoah-d2l) [reply components/consistent-evaluation.js:376]: It has to actually be a Uuid/Guid

File: components/controllers/ConsistentEvaluationHrefController.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { Classes, Rels } from 'd2l-hypermedia-constants';
 import { ConsistentEvaluationHrefControllerErrors } from './helpers/ConsistentEvaluationCommon.js';
 
 export class ConsistentEvaluationHrefController {
-	constructor(baseHref, token) {
+	constructor(baseHref, token, rubricWorkingCopyId) {
```
[the remark this answers is not in the document]
Comment (mpharoah-d2l) [reply components/controllers/ConsistentEvaluationHrefController.js:10]: It needs to use the same working copy id everywhere, so it needs to be stored to stay consistent

[the remark this answers is not in the document]
Comment (mpharoah-d2l) [reply components/controllers/ConsistentEvaluationHrefController.js:10]: Oh, wait, I see what you are saying. I'll see if that simplifies things at all

[the remark this answers is not in the document]
Comment (mpharoah-d2l) [reply components/controllers/ConsistentEvaluationHrefController.js]: Done

**Review by JamieTran — APPROVED** [review]

**Review by amauryperez — APPROVED** [review]

**Pull request merged**